manifests/chew-large-datasets.toml

```diff
@@ -0,0 +1,74 @@
+name = "chew-large-datasets"
+
+# hashicorp/go-getter URLs, so in the future we can support fetching test plans from GitHub.
+source_path = "file:${TESTGROUND_BASEDIR}/plans/chew-large-datasets"
+
+[build_strategies."docker:go"]
+enabled = true
+go_version = "1.13"
+module_path = "github.com/ipfs/testground/plans/chew-large-datasets"
+exec_pkg = "."
+  
+[build_strategies."exec:go"]
+enabled = true
+module_path = "github.com/ipfs/testground/plans/chew-large-datasets"
+exec_pkg = "."
+
+[run_strategies."local:docker"]
+enabled = true
+
+[run_strategies."local:exec"]
+enabled = true
+
+[run_strategies."cluster:nomad"]
+enabled = true
+
+[[testcases]]
+name = "ipfs-add-defaults"
```

Comment:
❓ I'm failing to replicate the dark magic that happens on the DHT manifest+plan in which a `lookup-peers` (https://github.com/ipfs/testground/blob/master/manifests/dht.toml#L28)  gets correctly set as an env var that it matches so that the test is selected at https://github.com/ipfs/testground/blob/master/plans/dht/main.go#L16 when `TESTGROUND_BASEDIR=`pwd` testground run dht/lookup-peers --builder=docker:go` is executed.

Reply:
Update: When running

```
> TESTGROUND_BASEDIR=`pwd` testground run chew-large-datasets/ipfs-add-defaults --builder=docker:go
```

I get

```
# ... docker logs
unknown runner:
```

What is this runner exactly?

Reply:
For reference:

- https://github.com/ipfs/testground/blob/master/pkg/engine/engine.go#L183-L186
- https://github.com/ipfs/testground/blob/master/pkg/engine/engine.go#L43-L44

Reply:
Nvm. It requires one more env var

```
> TESTGROUND_BASEDIR=`pwd` testground run chew-large-datasets/ipfs-add-defaults --builder=docker:go --runner="local:docker"
```

or

```
> TESTGROUND_BASEDIR=`pwd` testground run chew-large-datasets/ipfs-add-defaults --builder=docker:go --runner="local:exec"
```